lib/bgsite_official_web/router.ex

```diff
@@ -13,6 +14,14 @@ defmodule BgsiteOfficialWeb.Router do
     plug :fetch_current_admin
   end
 
+  pipeline :kaffy_browser do
```

Comment:
I think we can remove this snippet, it is already provided behind the scene by the above settings.